components/CandidateBigCard/index.js

```diff
@@ -49,28 +71,53 @@ const CandidateBigCard = (props) => {
               {capitalize(props.candidateFullname) || 'Nombre de candidato'}
             </Fullname>
             <Subtitle>
-              {props.district !== 'PERUANOS RESIDENTES EN EL EXTRANJERO'
-                ? `Postula por ${capitalize(props.district)}`
-                : `Postula al Extranjero`}
+              {contentPerCandidateType(
+                props.candidateRole,
+                props.district,
+                props.candidateGenre,
+              )}
             </Subtitle>
           </Column>
           <Row>
             <PartyIcon src={partyIconSource(props.candidateParty)} />
-            <NumberIcon>{props.candidateNumber || '5'}</NumberIcon>
+            {props.candidateRole.indexOf('CONGRESISTA') !== -1 ? (
+              <NumberIcon>{props.candidateNumber || 'NA'}</NumberIcon>
+            ) : null}
           </Row>
         </ContentColumn>
         <Star
           onClick={onStarClick}
           type={props.isFavorite ? 'favorite' : 'notFavorite'}
         />
       </MainRow>
-      <Row>
+      <RowBorderUp>
         {props.candidateDesignate === 'No' ? (
           <MemberBox type="green">Militante</MemberBox>
         ) : (
           <MemberBox type="yellow">Invitado del partido</MemberBox>
         )}
-      </Row>
+      </RowBorderUp>
+      {props.candidateTwitterLink || props.candidateFacebookLink ? (
+        <RowBorderUp>
+          <Subtitle>REDES SOCIALES</Subtitle>
+          {props.candidateTwitterLink ? (
+            <SocialLink href={props.candidateTwitterLink} target="_blank">
```

Comment:
No sé si `eslint` nos previene de esto out of the box (cc: @LuighiV), pero recuerdo que al tener un target="_blank" se recomienda tener un rel noreferrer, más info: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
sería así? rel="noopener noreferrer"

Reply:
lo puse así por ahora en el penúltimo commit

Reply:
pienso que sí 👍